src/main/java/org/sunbird/workflow/controller/WorkFlowController.java

```diff
@@ -112,7 +112,8 @@ public ResponseEntity<Response> getBulkUpdateStatus(@RequestHeader(Constants.X_A
 	}
 
 	@GetMapping(path = "/admin/bulkbuplodfile/download/{fileName}", produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<Response> downloadBulkuplodFile(@PathVariable("fileName") String fileName) {
+	public ResponseEntity<Response> downloadBulkuplodFile(@PathVariable("fileName") String fileName,
+														  @RequestHeader(Constants.X_AUTH_TOKEN) String userAuthToken) {
```

Comment:
Why thr userAuthToken is not used anywhere?

Reply:
removed